docs/user-guide/response-codes.md

```diff
@@ -1,15 +1,46 @@
 # Styx Response Codes
 
+Styx may return an HTTP error status code if it is unable to proxy a request.
+
+## Client Errors (4xx codes)
+
+When a client behaves in an unexpected way toward Styx, Styx will respond with specific error codes.
+This refers only to responses generated by Styx itself, not responses being proxied from downstream.
+For a general explanation of 4xx (Server Error) codes, please see the HTTP Spec.
+
+Note that interceptor plugins can also be programmed to modify response codes, so 
+they may introduce 4xx statuses for other reasons not listed here.
+
+
+### 400 Bad Request
+
+Occurs when the HTTP request format does not conform to HTTP 1.1 protocol specification, for example when 
+there is an unrecognised HTTP verb or multiple `Host` headers.
+
+
+### 408 Request Timeout
+
+While origin is client a HTTP Request, it must send something within `proxy.requestTimeoutMillis`.
+Any inactivity exceeding this will trigger a 408 response code. The timer applies only while the 
+HTTP request is being transmitted "over the wire". The timer is stopped when styx has fully 
+received request, so any delay or inactivity introduced by origin responding slowly will not 
+trigger this timeout.
```

Comment:
> "While origin is client a HTTP Request"

???

Reply:
Looks like I replaced the wrong word while editing.